prep-css-flexbox/display-flex.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
```

Comment:
Hi Andrew, looks like your HTML files didn't get save/added in the right order. Be sure to save them in VS Code before doing the `git add` step.

https://github.com/A-MAlmeida/prep-0721-code-solutions/pull/8/files